### PR TITLE
Refactor ClearCommand and Update DG ClearCommand

### PR DESCRIPTION
**Pull request by Weiennn**

### CODE REVIEW

File: docs/images/ClearCommandSequenceDiagram.png
Comment (ricketytoc) [inline docs/images/ClearCommandSequenceDiagram.png:1]: should deletePerson()

Comment (ricketytoc) [inline docs/images/ClearCommandSequenceDiagram.png:1]: Perhaps you might want to have () after deletePerson to show that it is a method

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Review by aexolate — APPROVED** [review]
LGTM

**Pull request merged**